static-site/app/styles/bootstrap.css

```diff
@@ -0,0 +1,3872 @@
+/*!
+ * Bootstrap Grid v4.6.2 (https://getbootstrap.com/)
+ * Copyright 2011-2022 The Bootstrap Authors
+ * Copyright 2011-2022 Twitter, Inc.
+ * Licensed under MIT (https://github.com/twbs/bootstrap/blob/main/LICENSE)
+ */
+html {
+  box-sizing: border-box;
+  -ms-overflow-style: scrollbar;
+}
+
+*,
+*::before,
+*::after {
+  box-sizing: inherit;
+}
+
+.container,
+.container-fluid,
+.container-sm,
+.container-md,
+.container-lg,
+.container-xl {
+  width: 100%;
+  padding-right: 15px;
+  padding-left: 15px;
+  margin-right: auto;
+  margin-left: auto;
+}
+
+@media (min-width: 576px) {
+  .container, .container-sm {
+    max-width: 540px;
+  }
+}
+
+@media (min-width: 768px) {
+  .container, .container-sm, .container-md {
+    max-width: 720px;
+  }
+}
+
+@media (min-width: 992px) {
+  .container, .container-sm, .container-md, .container-lg {
+    max-width: 960px;
+  }
+}
+
+@media (min-width: 1200px) {
+  .container, .container-sm, .container-md, .container-lg, .container-xl {
+    max-width: 1140px;
+  }
+}
```

Comment:
I noticed some additional whitespace on a wider screen and traced it back to the `container` class. The App router layout is respecting the bootstrap.css max-width of 1140px while the Pages router styled component `Container` has an additional @media rule 

https://github.com/nextstrain/nextstrain.org/blob/7617c384f3aa8fb3083c46a5c29f9f90db8c2558/static-site/src/components/splash/styles.tsx#L20-L22

Reply:
I'll have to dig and see where I got that 1500px rule from and how it's working on the current prod site...

Reply:
Ah, sorry I wasn't clear. I meant the 1500px rule is included in the current prod site but is not here.

Reply:
@joverlee521 Actually, [it is there](https://github.com/nextstrain/nextstrain.org/pull/1032/files#diff-414250c6938b686d9b4ef1135e2c3038098a52c8fb5d223792e2894c830a6226R49), unless I'm still misunderstanding you. 

I do see a layout difference on a larger screen, but it's only in the width of the divs starting with the team avatar icons -- from what I see, the layout from the top of the page down to and including "The core Nextstrain team is" matches exactly; stuff after that is clamped to 1140 ...

but yeah that's because the `@media(min-width: 1550px)` rule isn't cascading down from the `html` selector... 🤔

Reply:
ugh, nvm, stupid bootstrap.